ui-super-heroes/src/main/docker/Dockerfile.legacy-jar

```diff
@@ -0,0 +1,90 @@
+@@ -0,0 +1,89 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the container image run:
+#
+# ./mvnw package -Dquarkus.package.type=legacy-jar
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.legacy-jar -t quarkus/quinoa-app-legacy-jar .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/quinoa-app-legacy-jar
+#
+# If you want to include the debug port into your docker image
+# you will have to expose the debug port (default 5005) like this :  EXPOSE 8080 5005
+#
+# Then run the container using :
+#
+# docker run -i --rm -p 8080:8080 quarkus/quinoa-app-legacy-jar
```

Comment:
`quarkus/quinoa-app-jvm` _should_ be `quarkus/ui-super-heroes-jvm17` (to be consistent with all the other apps)